notebooks/transport/http.go

```diff
@@ -1,29 +1,26 @@
 package transport
 
 import (
-	"fmt"
 	"net/http"
-	"strconv"
 
 	"github.com/go-chi/chi"
 	"github.com/go-chi/chi/middleware"
-	"github.com/influxdata/influxdb/v2/context"
-	feature "github.com/influxdata/influxdb/v2/kit/feature"
+	"github.com/influxdata/influxdb/v2/kit/feature"
 	"github.com/influxdata/influxdb/v2/kit/platform"
 	"github.com/influxdata/influxdb/v2/kit/platform/errors"
 	kithttp "github.com/influxdata/influxdb/v2/kit/transport/http"
-	notebooks "github.com/influxdata/influxdb/v2/notebooks/service"
+	"github.com/influxdata/influxdb/v2/notebooks/service"
 	"go.uber.org/zap"
 )
 
 const (
-	prefixNotebooks = "/api/v2/notebooks"
+	prefixNotebooks = "/api/v2private/notebooks"
```

Comment:
This endpoint was recently updated (again) for the new `notebooksd` service. This is the endpoint that the current API requires. It should stay at `/api/v2private` for a while.